packages/react/src/feature-link-list/FeatureLinkListItem.tsx

```diff
@@ -0,0 +1,80 @@
+import { ReactNode } from 'react';
+import { LinkProps, packs } from '../core';
+import { TextLink, TextLinkExternal } from '../text-link';
+import { Box } from '../box';
+import { Flex } from '../flex';
+import { Stack } from '../stack';
+import { Text } from '../text';
+import { ArrowRightIcon } from '../icon';
+import { FeatureLinkListBackground, hoverColorMap } from './utils';
+
+export type FeatureLinkListItemProps = LinkProps & {
+	label: ReactNode;
+	description: ReactNode;
+	background?: FeatureLinkListBackground;
+};
+
+export const FeatureLinkListItem = ({
+	label,
+	description,
+	background = 'body',
+	...props
+}: FeatureLinkListItemProps) => {
+	const TextComponent = props.target == '_blank' ? TextLinkExternal : TextLink;
+
+	return (
+		<Box
+			as="li"
+			borderBottom
+			paddingBottom={0.25}
+			paddingTop={0.25}
+			css={{
+				'&:hover': {
+					backgroundColor: hoverColorMap[background],
+				},
+			}}
+		>
+			<Flex
+				gap={1.5}
+				paddingY={1}
+				background={background}
+				css={{
+					position: 'relative',
+					'&:hover': {
+						backgroundColor: hoverColorMap[background],
+					},
+					':has(:focus-visible)': packs.outline,
+					// If the browser does not support :has, use `:focus-within`
+					'@supports not selector(:has(*))': {
+						':focus-within': packs.outline,
+					},
+				}}
+				justifyContent="space-between"
+				alignItems="flex-start"
+			>
+				<Stack gap={0.5}>
+					<Text fontSize="md" fontWeight="bold">
+						<TextComponent
```

Comment:
I think it's worth adding `aria-describedby` pointing to the description

Reply:
Why do you say it doesn't work on interactive components? It's used a lot in the DS codebase